utility/test/module_test.go

```diff
@@ -60,23 +67,24 @@ func NewTestingUtilityContext(t *testing.T, height int64) utility.UtilityContext
 	}
 }
 
-// TODO(andrew): Take in `t` and fail the test if there's an error
-func newTestPersistenceModule(databaseUrl string) modules.PersistenceModule {
+func newTestPersistenceModule(_ *testing.M, databaseUrl string) modules.PersistenceModule {
 	cfg := modules.Config{
 		Persistence: &test_artifacts.MockPersistenceConfig{
 			PostgresUrl:    databaseUrl,
 			NodeSchema:     testSchema,
 			BlockStorePath: "",
 		},
 	}
-	// TODO(andrew): Move the number of actors into local constants
-	genesisState, _ := test_artifacts.NewGenesisState(5, 1, 1, 1)
+
+	genesisState, _ := test_artifacts.NewGenesisState(testingValidatorCount, testingServiceNodeCount, testingApplicationCount, testingFishermenCount)
```

Comment:
Check the error

Reply:
Not an error, please check function signature

Reply:
I actually did but forgot to delete the tentative comment. Not actionable.